src/sass/components/_button.scss

```diff
@@ -0,0 +1,15 @@
+.btn {
+	background-color: $background-color;
+	color: $color-primary;
+	padding: 0.5em 2em;
+	border: 1px solid $color-primary;
```

Comment:
Make it 0.15rem instead.

Reply:
Using `rem` for such small values don't make sense. It won't affect anything.